src/components/Header/Header.component.jsx

```diff
@@ -0,0 +1,79 @@
+import React, { useContext, useState } from 'react';
+import { AppContext } from '../../context/appContext';
+import { types } from '../../types/types';
+import { Toggle } from '../Toggle/Toggle.component';
+import {
+  ButtonIcon,
+  ButtonIconInput,
+  Form,
+  HiddenDown,
+  Menu,
+  MenuItem,
+  Navbar,
+  Spacer,
+  TextField,
+} from './Header.styled';
+
+export const Header = () => {
```

Comment:
Export this like so:
```
const Header = () => {...}

export default Header;
```

and in the `index.js` of this component file do this:
`export { default } from './Header.component'``

Then, you'll be able to import this component like so:
`import Header from 'components/Header'`